src/ontology/edits/oeo-social.omn

```diff
@@ -1225,28 +1225,34 @@ pull request https://github.com/OpenEnergyPlatform/ontology/pull/781",
 Class: OEO_00020076
 
     Annotations: 
-        OEO_00020426 "https://github.com/OpenEnergyPlatform/ontology/issues/256
-https://github.com/OpenEnergyPlatform/ontology/pull/800",
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/800
+make subclass of quantity value
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1917
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1931",
         <http://purl.obolibrary.org/obo/IAO_0000115> "A markup is an economic value that indicates a virtual amount added to an assumed or historical price to predict a price in a different context.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A markup is a quantity value that indicates a virtual amount added to an assumed or historical price to predict a price in a different context.",
```
As `economic value` gets deprecated and all become direct subclasses of quantity value, the definitions should be adapted (not only for this class, but all).